.github/workflows/integration-tests.yml

```
@@ -0,0 +1,261 @@
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        architecture: [hexagonal, layered, microservices]
+        ai_enabled: [true, false]
+        framework: [next, remix, expo]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
+
+      - name: Setup pnpm
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 39: could not find expected ':'

(syntax-check)